install_test/CMakeLists.txt

```diff
@@ -4,13 +4,11 @@ project(InstallTest LANGUAGES CXX)
 
 # `enable_language(CUDA)` is needed when ginkgo was statically build with CUDA
```

Comment:
Same here, I don't need this anymore on both CI and my Arch Linux when dealing with static libraries. I think we should be able to drop this in combination with the note in the [`README.md`](https://github.com/ginkgo-project/ginkgo/blob/develop/README.md#installing-ginkgo)